laborator/content/operatii-memorie-gdb/7-pointers/pointers.c

```diff
@@ -5,28 +5,73 @@
 
 #define SIGN(X) (((X) > 0) - ((X) < 0))
```

Comment:
Instead of the current commit subject please use:

`lab/memory-gdb: Fix asserts that compare pointers instead of data `.

Then at the end of commit add:.
Fixes: #125 


Thus when merging this PR, Gh will automatically close the isdue